src/ripple/app/tests/common_ledger.cpp

```diff
@@ -112,12 +112,11 @@ STTx
 parseTransaction(TestAccount& account, Json::Value const& tx_json, bool sign)
 {
     STParsedJSONObject parsed("tx_json", tx_json);
-    std::unique_ptr<STObject> sopTrans = std::move(parsed.object);
-    if (sopTrans == nullptr)
+    if (!parsed.object)
         throw std::runtime_error(
-        "sopTrans == nullptr");
-    sopTrans->setFieldVL(sfSigningPubKey, account.pk.getAccountPublic());
-    auto tx = STTx(*sopTrans);
+        "object not parseable");
```

Comment:
Quibble: this indentation is very strange, why not just glue it into the previous line?